backstop/tests/header.html

```diff
@@ -172,6 +172,29 @@ <h2>Default</h2>
                   aria-hidden="true"
                 ></svg-caret-down-small>
               </button>
+
+              <svg-chevron-right
+                class="iui-chevron"
+                aria-hidden="true"
+              ></svg-chevron-right>
+
+              <!-- Version Button -->
+              <button
```

Comment:
Why added this? Servers no purpose from what I can tell

Reply:
There is no test on button without dropdown. Otherwise, we would have noticed this inconsistency.
So I added this to have all possible cases shown.
I could either leave this or replace iModel picker or some other button with this.